json-schema/schema.json

```diff
@@ -243,6 +241,49 @@
           }
         }
       },
+      "if": {
+        "properties": {
+          "label:type": {
+            "const": "raster"
+          }
+        }
+      },
+      "then": {
+        "properties": {
+          "label:classes": {
+            "items": {
+              "properties": {
+                "name": {
+                  "oneOf": [
+                    {
+                      "type": "string",
+                      "minLength": 1
+                    },
+                    {
+                      "type": "null"
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        }
+      },
```

Comment:
Since the README says

> If labels are raster-formatted, use `null`.

I think it would make sense to collapse the `oneOf` down to just `"type": "null"`. I read that as a requirement that raster labels don't have a string `name` in their classes. Do you read it differently?

Reply:
I also read it like that... 

If label:type = raster => classes name = only null
If label:type = vector => classes name = only string

Reply:
Oops! Yes, definitely.